sycl/test-e2e/Basic/built-ins/marray_math.cpp

```diff
@@ -44,7 +44,7 @@
           cgh.single_task([=]() {                                              \
             sycl::multi_ptr<sycl::marray<PTR_TYPE, DIM>,                       \
                             sycl::access::address_space::global_space,         \
-                            sycl::access::decorated::yes>                      \
+                            sycl::access::decorated::no>                       \
```

Comment:
Note for reviewers: changed here to `no` just to check explicitly the case without decoration, and case with decoration will be checked implicitly here